config/README.md

```diff
@@ -27,6 +27,12 @@ This provides additional configuration support for end-to-end testing, including
 
 This configuration is used to generate `manifests/standard-e2e.yaml`.
 
+## config/overlays/prometheus
+
+Overlay containing manifest files which enable prometheus scraping of the catalogd and operator-controller pods. Used during e2e runs to measure performance over the lifetime of the test. 
+
+These manifests will not end up in the `manifests/` folder, as they must be applied in two distinct steps to avoid issues with applying prometheus CRDs and CRs simultaneously.
+
```

Comment:
Could we add here something like:

Note that the values defined for the performance checks are fixed and defined in : config/overlays/prometheus/prometheus_rule.yaml

Reply:
I am just trying to have it write down in some place to make easier understand it if we need afterwords